README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
+
+- Make it easy to build and package on all supported platforms.
+- Provide abstractions to avoid platform specific quirks.
+- Encourage good application behavior across platforms.
+- Automate platform resolution independence.
+- Enable robust internationalization.
+- Ensure accessibility to a wide group of people.
+- Produce small and fast binaries with low memory usage.
+- Have a small dependency tree with a high quality code base and good organization.
+- Focus on powerful, desktop-grade applications.
+- Provide the basic set of widgets needed by most applications.
```

Comment:
also a bit vague.
maybe: 

- Provide a flexible set of built-in widgets
- Make it easy to create custom widgets and application logic as needed

Reply:
Hmm, how about
- Offer a flexible set of layouts and common widgets.
- Ease creation of custom components and application logic as needed.